server/src/main/java/org/elasticsearch/index/fielddata/plain/LeafDoubleFieldData.java

```diff
@@ -25,15 +23,9 @@
  */
 public abstract class LeafDoubleFieldData implements LeafNumericFieldData {
 
-    private final long ramBytesUsed;
-
-    protected LeafDoubleFieldData(long ramBytesUsed) {
-        this.ramBytesUsed = ramBytesUsed;
-    }
-
     @Override
     public long ramBytesUsed() {
-        return ramBytesUsed;
+        return 0;
```

Comment:
Why some implementations return 0? Is this value ignored for some types?

Reply:
This was originally to handle un-inverted indices where we keep a ton of memory for the structures. In this case we really only have some fairly small references to lucene loaders so this is fairly safe.